python/arcticdb/version_store/library.py

```diff
@@ -201,6 +201,24 @@ class ReadRequest(NamedTuple):
     query_builder: Optional[QueryBuilder] = None
 
 
+class ReadInfoRequest(NamedTuple):
+    """ReadInfoRequest is designed to enable batching of read get descriptor / get metadata operations 
```

Comment:
`read get descriptor / get metadata operations` doesn't make sense. Say specifically which methods this is useful for.

Reply:
done